client/icecc-create-env.in

```diff
@@ -15,7 +15,7 @@ esac
 usage ()
 {
     echo "usage: $0 --gcc <gcc_path> <g++_path>"
-    echo "usage: $0 --clang <clang_path>"
+    echo "usage: $0 --clang <clang_path> [compiler_wrapper]"
     echo "usage: Use --addfile <file> to add extra files."
```

Comment:
This is intentionally omitted. The argument is deprecated and only for backwards compatibility.

Reply:
Unfortunately it interferes with --addfile. Right now if you want to use --clang and --addfile, you need to know that --clang slurps two arguments rather than one.

Reply:
That is the actual bug then. Fixed in 0943134df8eccaeb8c1fbb65a9088b8dea41a42c.